assets/src/containers/App.js

```diff
@@ -43,6 +44,7 @@ function App (props) {
               sideDrawerState={sideDrawerState}
               courseId={courseId}
               courseInfo={courseInfo} />
+            <GoogleAnalyticsTracking />
```

Comment:
This may not be the only tracking point to start with. With this PR[https://github.com/tl-its-umich-edu/my-learning-analytics/pull/535](url) we are adding the courseList page. I wonder if that get tracked at all with the change of [Router](https://github.com/tl-its-umich-edu/my-learning-analytics/pull/535/files#diff-e70b04386946903cb7c87b89a6de7cc0R3) setup.

Reply:
I've rebased the changes on #535 and refactored how the routers are setup. There should now only be one root level router (which GoogleAnalyticsTracking can be included in)